docs/cloud/ops/restore.md

```diff
@@ -0,0 +1,13 @@
+# Restore
```

Comment:
Is there a way to use the CLI tool? If not, we need to mention this.

Reply:
As I checked, it's possible by providing `backup_id` while creating the cluster. I created a separate issue for that https://github.com/EventStore/documentation/issues/319 to not block the whole PR.